aten/src/ATen/native/quantized/PackedParams.h

```diff
@@ -36,6 +36,30 @@ struct LinearPackedParamsBase : public torch::jit::CustomClassHolder {
     return output;
   }
 
+  // input is NOT quantized; quantization of input occurs within the function
+  // output is NOT quantized
+  virtual at::Tensor apply_with_input_q_dq_qweight_dq(
```

Comment:
can we make sure to include the ASCII graphs on every single c++ callsite added in this file to explain what this is doing?  This is not intuitive and I guarantee that most readers will have no idea what this function is doing based on the name, and unfortunately there isn't a way to easily navigate from here to the existing docblocks in aten/src/ATen/native/quantized/library.cpp

Reply:
sure, you mean just the two functions in this file right?

Reply:
sure, will change